cli/tests/integrations/test_analytics.py

```diff
@@ -115,64 +173,6 @@ def test_config_reporting_false():
         assert requests.post.call_count == 0
 
 
-@_mock
-def test_production_setting_true():
```

Comment:
We doesn't know exact reason, only think that this connected with some state on Windows. Fails in winapi function call: